miden-wasm/src/lib.rs

```diff
@@ -0,0 +1,86 @@
+use miden::{Assembler, ProgramInputs, ProofOptions};
+use wasm_bindgen::prelude::*;
+use wasm_bindgen_test::*;
+extern crate console_error_panic_hook;
+
+
+/// We derive Deserialize/Serialize so we can persist app state on shutdown.
+#[derive(serde::Deserialize, serde::Serialize)]
+pub struct InputFile {
+    pub stack_init: Vec<String>,
+    pub advice_tape: Option<Vec<String>>,
+}
+
+fn get_program_inputs(stack_init: &[u64], advice_tape: &[u64]) -> ProgramInputs {
```

Comment:
nit: It would be good to have doc comment for the method.

Reply:
What is a doc comment?

Reply:
https://doc.rust-lang.org/rust-by-example/meta/doc.html